### ISSUE TITLE
Problem in "mark_job_as_done" running GCP crawl

### ISSUE THREAD

**Issue by birdsarah** [mozilla/OpenWPM#624]
I have just run a gcp crawl of top 1000 sites.

docker image:  [commit-3959c67c38e88b986e842a2732fc099308f61d23](https://hub.docker.com/layers/openwpm/openwpm/commit-3959c67c38e88b986e842a2732fc099308f61d23/images/sha256-f6ec9ae4eec41e08c73800c9d8fae28e97862e0ef44d85b1319f732237763258?context=explore)

In doing so, I got 3 events in sentry with the following error.

```python
ValueError: list.remove(x): x not in list
  File "threading.py", line 864, in run
    self._target(*self._args, **self._kwargs)
  File "automation/TaskManager.py", line 374, in _mark_command_sequences_complete
    cs.mark_done()
  File "automation/CommandSequence.py", line 164, in mark_done
    self.callback()
  File "crawler.py", line 135, in mark_job_as_done
    unsaved_jobs.remove(job)
```

**Comment by birdsarah** [comment]
cc @vringar

**Comment by vringar** [comment]
This was hopefully fixed by #599
